### DISCUSSION TITLE
Welcome to multiple-cucumber-html-reporter Discussions!

### DISCUSSION THREAD

**Discussion by WasiqB** [WasiqB/multiple-cucumber-html-reporter#227] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BaCaRoZzo** [comment]
Hi @WasiqB, thanks for taking the mantle from Wim! Much appreciated.

I'm Federico and I'm a DevOps eng. who happens to co-maintain our internal testing suite. All the reporting of the suite is based on this project, which I took a bit at heart in the latest months. 🙂 

I think a very good use for the discussions is the definition of a roadmap. The project lacked one for quite some time and while we don't know all the challenges ahead, it would be cool to pin down what we think could be useful now or in the very near future.

**Reply by WasiqB** [reply]
I agree with you @BaCaRoZzo , we can put the features which we think can benefit the project and let the community have their say on the proposed features.

I am also thinking of having GitHub Projects setup with all the features that we have thought of, this will help us track everything.

🚨 BTW did you join our Discord server? If not, then please do so, it's a good place to hang out for the project community. Link is in the Readme.
